ODIN_II/regression_test/benchmark/task/syntax/synthesis_result.json

```diff
@@ -409,35 +409,23 @@
         "test_name": "syntax/bm_function_1/k6_frac_N10_frac_chain_mem32K_40nm",
         "architecture": "k6_frac_N10_frac_chain_mem32K_40nm.xml",
         "verilog": "bm_function_1.v",
-        "exit": 134,
-        "errors": [
-            "bm_function_1.v:40:1 [AST] Missing declaration of this symbol simple_func"
-        ]
+        "exit": 134
```

Comment:
the error changed here, and the missing declaration didn't move to warnings
can you check this?

Reply:
Here there actually isn't a missing declaration, if you look at the benchmark it's there so its good that it's now passing. It fails when it tries to pass a_in[`BITS-1:0] because the function it's in assumes they're both numbers and doesn't do a binary op check. I mentioned that this bm changed. I'm double checking tho